src/syft.js

```diff
@@ -7,159 +7,79 @@ import {
   WEBRTC_PEER_LEFT,
   WEBRTC_PEER_OPTIONS
 } from './_constants';
-import { NOT_ENOUGH_ARGS, NO_PLAN, PLAN_ALREADY_COMPLETED } from './_errors';
 
 import EventObserver from './events';
 import Logger from './logger';
 import Socket from './sockets';
 import WebRTCClient from './webrtc';
 import { protobuf, unserialize } from './protobuf';
+import GridAPIClient from './grid_api_client';
+import Job from './job';
+import SyftModel from './syft_model';
 
 export default class Syft {
   /* ----- CONSTRUCTOR ----- */
-  constructor({ url, verbose, workerId, scopeId, protocolId, peerConfig }) {
-    // My worker ID
-    this.workerId = workerId || null;
-
-    // The assigned scope ID
-    this.scopeId = scopeId || null;
-
-    // The chosen protocol we are working on
-    this.protocolId = protocolId || null;
-
-    // Our role in the protocol
-    this.role = null;
-
-    // The participants we will be working with (only for scope creators)
-    this.participants = [];
-
-    // The protocol we will are participating in
-    this.protocol = null;
+  constructor({ url, verbose, authToken, peerConfig }) {
+    // For creating verbose logging should the worker desire
+    this.logger = new Logger('syft.js', verbose);
 
-    // The plan we have been assigned
-    this.plan = null;
+    this.gridClient = new GridAPIClient({ url, logger: this.logger });
```

Comment:
Change the logger to not be included (we're switching to singleton): https://github.com/OpenMined/syft.js/issues/108